src/faro.ts

```diff
@@ -19,6 +19,7 @@ export enum FaroEvent {
   SHOW_TERRAFORM_CONFIG = 'show_terraform_config',
   REFETCH_TENANT_LIMITS = 'refetch_tenant_limits',
   INITIALIZE_ACCESS_TOKEN = 'initialize_access_token',
+  UPDATE_ALERTS = 'update_alerts',
```

Comment:
Unless "update_check_alerts" was factually wrong, it seems more descriptive than just "update_alerts" (assuming that we're talking about updating check alerts and not alerts in general).